forge-gui/res/cardsfolder/upcoming/mondrak_glory_dominus.txt

```diff
@@ -0,0 +1,10 @@
+Name:Mondrak, Glory Dominus
+ManaCost:2 W W
+Types:Legendary Creature Phyrexian Horror
+PT:4/4
+R:Event$ CreateToken | ActiveZones$ Battlefield | ValidToken$ Card.YouCtrl | ReplaceWith$ DoubleToken | Description$ If one or more tokens would be created under your control, twice that many of those tokens are created instead.
+SVar:DoubleToken:DB$ ReplaceToken | Type$ Amount | ValidCard$ Card.YouCtrl
+A:AB$ PutCounter | Cost$ 1 WP WP Sac<2/Artifact.Other;Creature.Other/other artifacts and/or creature> | Defined$ Self | CounterType$ Indestructible | CounterNum$ 1 | SpellDescription$ Put an indestructible counter on CARDNAME.
```

Comment:
does this cost look right in Card Detail, or should it end with `creatures>` ?

Reply:
Add the reminder text to SpellDescription... auto stack description should drop it